app/bundles/StageBundle/Tests/Unit/EventListener/CampaignSubscriberTest.php

```diff
@@ -49,6 +51,7 @@ public function __construct()
             {
             }
 
+            /** @phpstan-ignore-next-line */
```

Comment:
Please add to the comment why was the ignore necessary.

Reply:
added a comment